lib/rubocop/cop/rails/duplicate_association.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop looks for associations that have been defined multiple times in the same file.
```

Comment:
How about adding to the documentation that auto-correction uses the last defined association in duplicated same associations for compatibility?

Reply:
Done! Let me know what you think.